src/abaqus/Feature/Feature.py

```diff
@@ -1518,7 +1518,7 @@ def PartitionCellByPlaneNormalToEdge(
     def PartitionCellByPlanePointNormal(
         self,
         cells: Sequence[Cell],
-        point: Union[ConstrainedSketchVertex, InterestingPoint, DatumPoint],
+        point: Union[ConstrainedSketchVertex, InterestingPoint, DatumPoint, Datum],
```

Comment:
Why a Datum object can be accepted?

Reply:
It is a way to remove this type warning

![image](https://user-images.githubusercontent.com/57204023/195864581-873e0394-e402-4273-9418-d1b658e2d06d.png)

Reply:
![image](https://user-images.githubusercontent.com/57204023/195864677-a834ec81-0c49-4f9e-a2ee-9819c1039849.png)

Reply:
But I think it is not really correct to do that

Reply:
I reverted that change

Reply:
Maybe there are some inheritance errors between these classes

Reply:
But we use dictionaries instead of repositories in the typing annotations, VSCode should able to detect types of the objects correctly.

Reply:
Yeah, the type detection is correct. But in this case, for example, the repository is
```python
mdb.models['Model-1'].parts['blockE'].datums
```
And the element it is trying to access is the `mdb.models['Model-1'].parts['blockE'].datums[2]`

But there is no restriction of this element to be a `DatumPoint`, because its definition is the general type `Datum`

![image](https://user-images.githubusercontent.com/57204023/195874159-56a7a7cb-4a55-43ac-b095-9a2ad109e4f1.png)

The restriction of being a `DatumPoint` is when we create the object.

Reply:
So the`DatumPoint` class should be inherited from `Datum`, in this way a `DatumPoint` object is also a `Datum` object.

Reply:
> So the `DatumPoint` class should be inherited from `Datum`, in this way a `DatumPoint` object is also a `Datum` object.

That's correct. But the opposite is not true: a `Datum` object is not a `DatumPoint` object. That is the reason of the warning, I think.

Reply:
> > So the `DatumPoint` class should be inherited from `Datum`, in this way a `DatumPoint` object is also a `Datum` object.
> 
> That's correct. But the opposite is not true: a `Datum` object is not a `DatumPoint` object. That is the reason of the warning, I think.

That is why `DatumPoint`should be inherited from `Datum` rather than the opposite.

Reply:
> That is why `DatumPoint` should be inherited from `Datum` rather than the opposite.

I didn't understand. Is not that what is implemented?
```python
# content of DatumPoint.py
class DatumPoint(Datum):
    """The DatumPoint object has no direct constructor; it is created when a Feature object is
    created. For example, the DatumPointByCoordinate method creates a Feature object that
    creates a DatumPoint object.
    The DatumPoint object is derived from the Datum object.
    ....
    """
```

Reply:
Oh, I think it in the wrong way.

Reply:
This should be fine, we may just ignore the warning

Reply:
Yeah, I think so.

Reply:
Maybe your first idea is quite good.

Reply:
It solved this particular warning, but maybe it was not correct.

Reply:
Yes, let's keep it in the original way